components/ProfilePage/AboutMeEditForm.tsx

```diff
@@ -28,22 +32,16 @@ async function updateProfile(
   { profile, actions, uid }: Props,
   data: UpdateProfileData
 ) {
-  const {
-    updateIsPublic,
-    updateSocial,
-    updateAbout,
-    updateDisplayName,
-    updateProfileImage,
-    updateIsOrganization
-  } = actions
+  const { updateIsPublic, updateSocial, updateAbout, updateDisplayName } =
+    actions
 
   await updateIsPublic(data.public)
-  await updateSocial("linkedIn", data.linkedIn)
-  await updateSocial("twitter", data.twitter)
-  await updateAbout(data.aboutYou)
-  await updateDisplayName(data.name)
-  await updateIsOrganization(data.organization)
-  // await updateProfileImage(data.profileImage) disabled until permissions to be updated in fb storage
+  data.linkedIn && (await updateSocial("linkedIn", data.linkedIn))
+  data.twitter && (await updateSocial("twitter", data.twitter))
+  data.aboutYou && (await updateAbout(data.aboutYou))
+  data.name && (await updateDisplayName(data.name))
+
+  console.log(profile)
```

Comment:
If these are empty, we should delete the field. You may need to add `?? deleteField()` to some of the update functions in profile.ts (see others in that file)